Client/Frontend/Browser/TabManager.swift

```diff
@@ -174,6 +176,10 @@ class TabManager : NSObject {
 
     // This method is duplicated to hide the flushToDisk option from consumers.
     func removeTab(tab: Browser) {
+        if tabsNetworkActivity[tab.webView!] == true {
+            decrementNetworkActivityCount()
+        }
+        tabsNetworkActivity.removeValueForKey(tab.webView!)
```

Comment:
I would not assume this is true. We lazily create the webviews on startup for session restore. Its possible this tab was never touched.